runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -523,6 +523,7 @@
 	SEED_VAULT_ITEM_TEXT(WidgetID.SEED_VAULT_GROUP_ID, WidgetID.SeedVault.ITEM_TEXT),
 	SEED_VAULT_INVENTORY_ITEMS_CONTAINER(WidgetID.SEED_VAULT_INVENTORY_GROUP_ID, WidgetID.SeedVault.INVENTORY_ITEM_CONTAINER),
 
+	OPTIONS_CAMERA_ZOOM_SLIDER_BUTTON(WidgetID.OPTIONS_GROUP_ID, WidgetID.Options.CAMERA_ZOOM_SLIDER_BUTTON),
```

Comment:
`OPTIONS_CAMERA_ZOOM_SLIDER_HANDLE` would be more in line with how we describe slider handles in, say, the music plugin.